### PR TITLE
Matts moves

### LINKED ISSUE #2081: Create new page: Manage access
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

New page

### What part(s) of the page would you like to see updated?

Create an overview page that describes access management for Core and Cloud.

### Additional information

_No response_

### LINKED ISSUE #2082: Create new page:  Build and view your docs with dbt Cloud
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

- Generating documentation in dbt Cloud -- (https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-generating-documentation)
- Viewing docs in the IDE -- https://docs.getdbt.com/docs/dbt-cloud/cloud-ide/viewing-docs-in-the-ide

### What part(s) of the page would you like to see updated?

- Merge the [Generating documentation in dbt Cloud](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-generating-documentation) page with the [Viewing docs in the IDE](https://docs.getdbt.com/docs/dbt-cloud/cloud-ide/viewing-docs-in-the-ide) page
- After the page merge, name the new page: Build and view your docs with dbt Cloud
- Remove the redundant file(s) and create redirects 

### Additional information

_No response_

### LINKED ISSUE #2083: Create new page: Upgrade Core version in Cloud
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

- Upgrading dbt versions in Cloud -- https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions
- Choosing a dbt version -- https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-choosing-a-dbt-version

### What part(s) of the page would you like to see updated?

- Rename this page https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions to "Upgrade Core version in Cloud"
- Add contents to this ^ page from this https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-choosing-a-dbt-version
- Delete the page https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-choosing-a-dbt-version and create a redirect 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by matthewshaver**
## Checklist
If you added new pages (delete if not applicable):
- [ x] The page has been added to `website/sidebars.js`
- [ x] The new page has a unique filename

### CODE REVIEW

File: website/docs/docs/collaborate/cloud-build-and-view-your-docs.md [hunk 1]
```diff
@@ -40,3 +40,23 @@ Once you have a job set up to generate documentation for your project, then clic
 ## Easy access
 
 See [Building and Configuring Artifacts](artifacts) for more information on exposing links to the latest Documentation and Source Freshness reports to your team.
+
+## Viewing docs in the IDE
```
Comment (runleonarun) [inline website/docs/docs/collaborate/cloud-build-and-view-your-docs.md]:
```suggestion
## Viewing docs
```
This section and the one above describe the same thing. Can you combine them under this heading?

> ## Viewing documentation
> Once you have a job set up to generate documentation for your project, then click **Documentation** in the top left. Your project's documentation should open. This link will always navigate you to the most recent version of your project's documentation in dbt Cloud.
<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/98c05c5-Screen_Shot_2019-02-08_at_9.18.22_PM.png" title="Project documentation"/>

File: website/docs/docs/collaborate/cloud-build-and-view-your-docs.md [hunk 2]
```diff
@@ -40,3 +40,23 @@ Once you have a job set up to generate documentation for your project, then clic
 ## Easy access
```
Comment (runleonarun) [inline website/docs/docs/collaborate/cloud-build-and-view-your-docs.md]:
```suggestion
## Provide access
```

File: website/docs/docs/collaborate/manage-access/manage-access.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
+---
+title: "Manage access"
+id: "manage-access"
```
Comment (nghi-ly) [inline website/docs/docs/collaborate/manage-access/manage-access.md]: should this be "About access" ?

File: website/docs/docs/collaborate/cloud-build-and-view-your-docs.md [hunk 4]
```diff
@@ -40,3 +40,23 @@ Once you have a job set up to generate documentation for your project, then clic
...
 
 See [Building and Configuring Artifacts](artifacts) for more information on exposing links to the latest Documentation and Source Freshness reports to your team.
```
Comment (runleonarun) [inline website/docs/docs/collaborate/cloud-build-and-view-your-docs.md]:
```suggestion
Before the documentation can be generated, you need to select the job to create the artifacts.  Read more in [Building and Configuring Artifacts](artifacts).
```

File: website/docs/docs/collaborate/cloud-build-and-view-your-docs.md [hunk 5]
```diff
@@ -40,3 +40,23 @@ Once you have a job set up to generate documentation for your project, then clic
...
+
+## Viewing docs in the IDE
+
+The dbt Cloud IDE makes it possible to view [documentation](/building-a-dbt-project/documentation)
```
Comment (runleonarun) [inline website/docs/docs/collaborate/cloud-build-and-view-your-docs.md]:
```suggestion
The dbt Cloud IDE makes it possible to view [documentation](/docs/collaborate/documentation)
```

**Review by runleonarun — CHANGES_REQUESTED** [review]
@matthewshaver Just a few tweaks and this will be good!

File: website/sidebars.js [hunk 6]
```diff
@@ -264,7 +264,7 @@ const sidebarSettings = {
           type: "category",
           label: "Manage access",
           items: [
-            "docs/collaborate/manage-access/licenses-and-groups",
+            "docs/collaborate/manage-access/manage-access",
```
Comment (matthewshaver) [inline website/sidebars.js]:
```suggestion
            "docs/collaborate/manage-access/about-access",
```

File: website/docs/docs/collaborate/manage-access/about-access.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+Permission sets are predefined collections of granular permissions. Permission
+sets combine low-level permission grants into high-level roles that can be
+assigned to groups. Some examples of existing permission sets are:
+ - Account Admin
+ - Git Admin
+ - Job Admin
+ - Job Viewer
+ - ...and more
+
+For a full list of enterprise permission sets, see [Enterprise Permissions](/docs/dbt-cloud/access-control/enterprise-permissions).
```
Comment (runleonarun) [inline website/docs/docs/collaborate/manage-access/about-access.md]:
```suggestion
For a full list of enterprise permission sets, see [Enterprise Permissions](/docs/collaborate/manage-access/enterprise-permissions).
```

File: _redirects [hunk 8]
```diff
@@ -12,7 +12,7 @@
 /docs/dbt-cloud/access-control/enterprise-permissions /docs/collaborate/manage-access/enterprise-permissions 301
 /docs/dbt-cloud/access-control/self-service-permissions /docs/collaborate/manage-access/self-service-permissions 301
 /docs/dbt-cloud/access-control/cloud-seats-and-users /docs/collaborate/manage-access/seats-and-users 301
-/docs/dbt-cloud/access-control/access-control-overview /docs/collaborate/manage-access/licenses-and-groups 301
+/docs/dbt-cloud/access-control/access-control-overview /docs/collaborate/manage-access/manage-access 301
```
Comment (matthewshaver) [inline _redirects]:
```suggestion
/docs/dbt-cloud/access-control/access-control-overview /docs/collaborate/manage-access/about-access 301
```

**Review by runleonarun — CHANGES_REQUESTED** [review]
One more change!

File: website/docs/docs/collaborate/manage-access/about-access.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+invited to a given account. This license type may change over time, but a
+user can only have one type of license at any given time.
+
+A user's license type controls the features in dbt Cloud that the user is able
+to access. dbt Cloud's two license types are:
+ - **Read Only**
+ - **Developer**
+
+For more information on these license types, see [Seats & Users](cloud-seats-and-users).
+At a high-level, Developers may be granted _any_ permissions, whereas Read Only
```
Comment (runleonarun) [inline website/docs/docs/collaborate/manage-access/about-access.md]:
```suggestion
At a high level, Developers may be granted _any_ permissions, whereas Read Only
```

**Review by runleonarun — APPROVED** [review]
This looks good now!

**Pull request merged**